security/putty/Portfile

```diff
@@ -19,28 +20,13 @@ homepage            http://www.chiark.greenend.org.uk/~sgtatham/putty/
 master_sites        ftp://ftp.chiark.greenend.org.uk/users/sgtatham/putty-${version}/ \
                     http://the.earth.li/~sgtatham/putty/${version}/
 
-checksums           rmd160  9357dfca867dc91be09c735b62025f9c614ab46d \
-                    sha256  bb8aa49d6e96c5a8e18a057f3150a1695ed99a24eef699e783651d1f24e7b0be
-
-worksrcdir          ${worksrcdir}/unix
+checksums           rmd160  d4e27e59e3a7235089ccae854db2507a650da02f \
+                    sha256  419a76f45238fd45f2c76b42438993056e74fa78374f136052aaa843085beae5 \
+                    size    2619481
 
 depends_build       port:pkgconfig
```

Comment:
you should use here `depends_build-append` and below as well `depends_lib-append` as not to overwrite any of the dependencies added by the PortGroup. As you can see in the CI now the build fails because it cannot find `cmake`; that build dependency is added by the cmake PG but you effectively remove it here again.

Reply:
Oh. Fixed